packages/cdktf-cli/build.ts

```diff
@@ -50,11 +50,7 @@ const nativeNodeModulesPlugin = {
 
 (async () => {
   await esbuild.build({
-    entryPoints: [
-      "./bin/cdktf.ts",
-      "./lib/server/server.ts",
-      "./bin/cmds/handlers.ts",
-    ],
+    entryPoints: ["./bin/cdktf.ts", "./bin/cmds/handlers.ts"],
```

Comment:
This one line change has a really weird outcome

### Before the change

```
packages/cdktf-cli/bundle/
├── bin
│   ├── cdktf
│   ├── cdktf.js
│   ├── cdktf.js.map
│   └── cmds
│       ├── handlers.js
│       └── handlers.js.map
├── fsevents-72LCIACT.node
└── lib
    └── server
        ├── schema.graphql
        ├── server.js
        └── server.js.map
```

### After the change

```
packages/cdktf-cli/bundle/
├── bin
│   └── cdktf
├── cdktf.js
├── cdktf.js.map
├── cmds
│   ├── handlers.js
│   └── handlers.js.map
└── lib
    └── server
        └── schema.graphql

4 directories, 6 files
```

Reply:
Adding the lib/index.ts (API) helped to get into the old format, we could also change the paths used in the cdktf binary file, but this seemed less intrusive.

Reply:
Weird 😅 Happy with either solution 😁